flask_monitoringdashboard/frontend/js/controllers/configuration.js

```diff
@@ -1,14 +1,101 @@
-export function ConfigurationController($scope, $http, menuService, endpointService) {
+export function ConfigurationController($scope, $http, menuService, endpointService, modalService) {
     endpointService.reset();
     menuService.reset('configuration');
+    modalService.setConfirm('delete', () => deleteUser($scope.user));
```

Comment:
Bit of an awkward API if you ask me. A result of (if you ask me) a bad generalisation of modals :) (see below)